docs/OCP_CI_Tutorials/Scenarios/Execution_Guide.md

```diff
@@ -16,9 +18,11 @@ We define the Execution step of a scenario as:
 Please follow the guide below to create the execution step of your scenario.
```

Comment:
Given that you capitalize "Execution" earlier in this document when discussing the Execution step, we should be consistent here.

Reply:
```suggestion
Please follow the guide below to create the Execution step of your scenario.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)